aws/requests.yaml

```
@@ -13,6 +13,10 @@ releases:
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: "> 13.1.0 > 14.0.0 > 14.1.0"
```

Shall we just name it `> 13.1.0` ?
```suggestion
    - name: "> 13.1.0"
```

---

My intent was to avoid it involving existing 14.0.x and 14.1.x releases but I probably didn't fulfil it anyway

---

Ah I see 👍🏻 , all good